.github/actions/common/unzip-all-zips/action.yaml

```diff
@@ -0,0 +1,39 @@
+name: "Unzip all zips"
+description: "Unzip all the zip files located in a given directory"
+inputs:
+  input-dir:
+    description: 'Directory that contains zip files'
+    required: true
+  output-dir:
+    description: 'Output directory that will contain all unzipped files'
+    required: true
+  include-pattern:
+    description: 'Remove all files that does not fit this pattern'
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - name: Create Output dir
+      id: create-output-dir
+      shell: bash
+      run: |
+        mkdir ${{ inputs.output-dir }}
+
+    - name: unzip all files
+      id: unzip
+      shell: bash
+      run: |
+        for zip_file in ${{ inputs.input-dir }}/*.zip; do
+          if [[ -e "$zip_file" ]]; then
+            unzip -o "$zip_file" -d ${{ inputs.output-dir }}
+          fi
+        done
+
+    - name: Filter Include Pattern
+      id: filter-include-pattern
```

Comment:
Do we need an `id` for each of these steps? I think those are only needed if we're exporting a `GITHUB_OUTPUT` variable.

Reply:
Originally, there was an intention for using some outputs but it has been reverted. Since specifying an id doesn't have any side effect, I left it for possible future use. If you think it's unnecessary I can remove it

Reply:
If there isn't currently a use for it then let's remove it. We can always add ids in the future if we need them.